.gitignore

```diff
@@ -2,4 +2,5 @@ __pycache__/
 */__pycache__/
 /data
 /runs
-poc.py
```

Comment:
can we stop tracking __pycache files, see link below on clearing git cache. https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
also can we stop tracking /runs. I suspect you have to clear cache, see link attached above.

Reply:
Resolved.